docs/source/alfacase/03_alfacase_example.rst

```diff
@@ -0,0 +1,121 @@
+.. _alfacase-example:
+
+ALFAcase Example
+================
+
+In this section, we will create a project from scratch with a simple network and insert it on the ALFAsim application.
+
+
+The root level of our :program:`ALFAcase` file is always a :class:`CaseDescription` which has the following attributes:
+For the full reference check the :ref:`alfacase-reference-section`
+
+.. code-block:: yaml
+
+    name: string    # optional
+    physics: physics_description_schema⠀
+    time_options: time_options_description_schema⠀
+    numerical_options: numerical_options_description_schema⠀
+    ipr_models: ipr_models_description_schema⠀
+    pvt_models: pvt_models_description_schema⠀
+    tracers: tracers_description_schema⠀
+    outputs: case_output_description_schema⠀
+
+    ....
+
+We are going to create a simple case with a PVT model, two nodes and one pipe displaying their output.
+Each section can be inspect over the :ref:`alfacase-reference-section`
+
+.. code-block:: yaml
+
+    name: basic_case
+    physics:
+    correlations_package: correlation_package_alfasim
+    hydrodynamic_model: hydrodynamic_model_4_fields
+
+    pvt_models:
+    default_model: 'Pvt1'
+    tables:
+      'Pvt1': my_pvt_file.tab
+
+    numerical_options:
+    tolerance: 0.0001
+
+    time_options:
+    minimum_timestep:
```

Comment:
These should be indented, right?

Reply:
yep, probably I modified by mistake this section after running this snippet through a "prettify code" tool.